openquake/sep/tests/test_sep_suite_2.py

```diff
@@ -103,18 +148,168 @@ def test_hazus_liquefaction_prob(self):
+
+    def test_allstadt_2022(self):
+        prob_liq, out_class, LSE = allstadt_etal_2022(
+            pgv=self.pgv, pga=self.pga, mag=self.mag, vs30=self.sites["vs30"], dw=self.sites["dw"], wtd=self.sites["gwd"], 
+            precip=self.sites["precip"])
+
+        zlp = np.array([0.235711715, 0.133029207, 0.321699203, 0.25027888, 0.342388975,
+                        0.454210576, 0.33122703, 0, 0.355359841, 0.249738624])
+        
+        clq = np.array([0, 0, 0, 0, 0, 1, 0, 0, 0, 0])
+
+        lse = np.array([1.417553345, 0.268584431, 4.732082598, 1.764759601, 6.098616151,
+                        17.83890389, 5.329037755, 0.026094205, 7.090862255, 1.750645015])
+        
+        self.sites["allstadt22_liq_prob"] = prob_liq
+        self.sites["allstadt22_liq_class"] = out_class
+        self.sites["allstadt22_liq_se"] = LSE
+
+        np.testing.assert_array_almost_equal(self.sites["allstadt22_liq_prob"], zlp)
+        np.testing.assert_array_almost_equal(self.sites["allstadt22_liq_class"], clq)
+        np.testing.assert_array_almost_equal(self.sites["allstadt22_liq_se"], lse)
+
+    def test_akhlagi_2021_model_a(self):
+        prob_liq, out_class = akhlagi_etal_2021_model_a(
+            pgv=self.pgv, tri=self.sites["tri"], dc=self.sites["dc"], dr=self.sites["dr"], 
+            zwb=self.sites["zwb"])
+
+        zlp = np.array([0.949740005, 0.660621806, 0.982407491, 0.972949802, 0.989203293,
+                        0.992850986, 1.58769E-05, 0.526810919, 0.988698807, 0.908844299])
+        
+        clq = np.array([1, 1, 1, 1, 1, 1, 0, 1, 1, 1])
+
+        self.sites["akhlagi21_prob_a"] = prob_liq
+        self.sites["akhlagi21_class_a"] = out_class
+
+        np.testing.assert_array_almost_equal(self.sites["akhlagi21_prob_a"], zlp)
+        np.testing.assert_array_almost_equal(self.sites["akhlagi21_class_a"], clq)
+        
+    def test_akhlagi_2021_b(self):
+        prob_liq, out_class = akhlagi_etal_2021_model_b(
+            pgv=self.pgv, vs30=self.sites["vs30"], dc=self.sites["dc"], dr=self.sites["dr"], 
+            zwb=self.sites["zwb"])
+
+        zlp = np.array([0.973289334, 0.974526031, 0.988384878, 0.988594348, 0.990864928,
+                        0.992975958, 0.991752215, 0.989183352, 0.992783854, 0.990591401])
+        
+        clq = np.array([1, 1, 1, 1, 1, 1, 1, 1, 1, 1])
+        
+        self.sites["akhlagi21_prob_b"] = prob_liq
+        self.sites["akhlagi21_class_b"] = out_class
+
+        np.testing.assert_array_almost_equal(self.sites["akhlagi21_prob_b"], zlp)
+        np.testing.assert_array_almost_equal(self.sites["akhlagi21_class_b"], clq)
+
+    def test_todorovic_2022(self):
+        out_class, out_prob = todorovic_silva_2022_nonparametric_general(
+            pgv=self.pgv, vs30=self.sites["vs30"], dw=self.sites["dw"], wtd=self.sites["gwd"], 
+            precip=self.sites["precip"])
+
+        self.sites["todorovic22_liq_class"] = out_class  
+        self.sites["todorovic22_liq_prob"] = out_prob
```

Comment:
Why setting this? In the check below you could just use out_class and out_prob

Reply:
Removed as suggested